rust/hyperlane-base/src/settings/chains.rs

```diff
@@ -21,14 +20,27 @@ use crate::CoreMetrics;
 ///
 /// Specify the chain name (enum variant) in toml under the `chain` key
 #[derive(Clone, Debug, Deserialize)]
-#[serde(tag = "rpcStyle", content = "connection", rename_all = "camelCase")]
+#[serde(
+    tag = "implementation",
+    content = "connection",
+    rename_all = "camelCase"
+)]
 pub enum ChainConf {
     /// Ethereum configuration
-    Ethereum(ConnectionConf),
+    Ethereum(h_eth::ConnectionConf),
```

Comment:
Why h_eth::ConnectionConf and not just ConnectionConf?

Reply:
This got split off from #1474 and it _will_ make sense once we have hyperlane eth and hyperlane fuel.